client/extensions/woocommerce/app/settings/payments/payment-method-row.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import TableRowField from '../../../components/table/table-row-field';
+
+const PaymentMethodRow = ( { method, translate } ) => {
+	return (
+		<tr className="payments__method-row">
+			<TableRowField>
+				{
+					method.isSuggested &&
+					(
+						<p className="payments__method-suggested">
+							{ translate( 'Suggested Method' ) }
+						</p>
+					)
+				}
+				<p>{ method.label }</p>
+			</TableRowField>
+			<TableRowField>
+				<p>{ method.fee }</p>
+				<p>
+					<a href={ method.information }>
+						{ translate( 'More Information' ) }
+					</a>
+				</p>
+			</TableRowField>
+			<TableRowField className="payments__setup-column">
+				<Button compact>
+					{
+						translate( 'Set Up' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Set up' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).